appengine/standard/migration/taskqueue/pull-counter/.gcloudignore

```diff
@@ -0,0 +1,19 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
Is it typical to include a .gcloudignore with our samples? Or is it like a .gitignore where we don't need to bother committing it?

Reply:
It varies, but in this case it was an oversight on my part. I've taken it out of the next commit.